tests/test_processors_ayon_hiero_effect_file.py

```diff
@@ -120,3 +121,57 @@ def test_EffectsFileProcessor(self, kwargs, results):
         assert (
             len(processor.ocio_objects) == results["expected_length"]
         )
+
+    def test_clear_operators(self):
+        effect_processor = AYONHieroEffectsFileProcessor(
+            Path("dummy.json")
```

Comment:
seems this `dummy.json` isn't available in the repo.
https://github.com/ynput/LabLib/actions/runs/12292170415/job/34302276633#step:9:272

let's push this into `resources/public`